README.md

```diff
@@ -51,7 +51,7 @@ Next, lets configure the development server. To keep things simple we can put th
 var hypernova = require('hypernova/server');
 
 hypernova({
-  devMode: true,
+  clustering: false,
```

Comment:
By default it's recommended that you run Hypernova with clustering enabled. What sort of setup are you running in production and what would you typically expect?